helpers/constants.js

```diff
@@ -118,8 +98,6 @@ var constants = {
 		// Testnet
 		'da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
 	],
-	numberLength: 100000000,
```

Comment:
`numberLength` should be renamed, kept and used on test suites. We already have the same concept named `normalizer` instead of getting it from constants. I suggest to rename and get it in the test suite from here https://github.com/LiskHQ/lisk/blob/02d35b04afb2384db1e1b4257b0b8d202ea3fda5/test/common/utils/normalizer.js#L17